mountinfo/mountinfo_linux.go

```diff
@@ -194,7 +194,7 @@ func unescape(path string) (string, error) {
 			v := c - '0'
 			for j := 2; j < 4; j++ { // one digit already; two more
 				x := s[j] - '0'
-				if x < 0 || x > 7 {
+				if x > 7 {
```

Comment:
OK I think the correct thing to do would be
```go
if s[j] < '0' || s[j] > '7' {
  return "", fmt.Errorf("bad escape sequence %q: not a digit", s[:3])
}
x := s[j] - '0'
```

Reply:
This won't fix anything (we will still catch underflow in case s[j] is less than '0'), but it's a tad cleaner code with no ambiguity.